src/source/Main.scala

```diff
@@ -282,6 +285,15 @@ object Main {
     } else {
       None
     }
+    val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderOut.isDefined) {
```

Comment:
I'm not sure I follow.  All the swift-bridging-header generation seems to happen all at once inside of SwiftBridgingHeaderGenerator.generate().  This long-lived writer would make sense if the bridging header was being output piecemeal as a side-effect of ObjC generation, similar to what's done with the `outFileListWriter`.  With the current implementation, it seems like it could be handled entirely inside of SwiftBridgingHeaderGenerator.generate().

Regardless, my prior comment was less about the implementation than about the model from a user perspective.  There's extra complexity if the objc and swift-bridging-header generators can be enabled separately, and given different output directories (the latter is why you need to create the directory below).  I feel like it might be simpler for the user to see the swift bridging header as simply an optional output of the ObjC generator, which they simple enable optionally, and provide with a filename (not a path).